libjulia/backends/evm/EVMCodeTransform.h

```diff
@@ -66,13 +67,23 @@ class CodeTransform: public boost::static_visitor<>
 	{
 	}
 
+	CodeTransform& initReferencesCounter(Block const& _block)
+	{
+		ReferencesCounter counter;
```

Comment:
This can be shortened to `m_references = ReferencesCounter(_block).references();`